editor/icons/EditAddRemove.svg

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Editor icons should be optimized IIRC. https://docs.godotengine.org/en/latest/development/editor/creating_icons.html#icon-optimization